tensorflow/contrib/rnn/BUILD

```diff
@@ -26,7 +26,8 @@ load(
 
 tf_custom_op_py_library(
     name = "rnn_py",
-    srcs = ["__init__.py"] + glob(["python/ops/*.py"]),
+    srcs = ["__init__.py"] + glob(["python/ops/*.py"]) 
+                           + glob(["python/tools/*.py"]),
```

Comment:
How about this instead:
  srcs = ["__init__.py", "python/tools/checkpoint_convert.py"] + glob(["python/ops/*.py"])
This excludes the test by not using the glob, instead of moving the test into its own directory (creating two directories with one file each).